spring-integration-file/src/main/java/org/springframework/integration/file/remote/synchronizer/AbstractInboundFileSynchronizer.java

```diff
@@ -227,6 +228,11 @@ public void close() throws IOException {
 
 	@Override
 	public void synchronizeToLocalDirectory(final File localDirectory) {
+		synchronizeToLocalDirectory(localDirectory, Integer.MIN_VALUE);
```

Comment:
We could do that, but we really have apply the max after the filter because, unless the remote files is deleted, we'll never fetch any more files (with AcceptOnceFLF).